packages/fxa-content-server/app/scripts/lib/glean/registration.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// AUTOGENERATED BY glean_parser v6.2.1. DO NOT EDIT. DO NOT COMMIT.
```

Comment:
Generated, yes.  Do not commit? nah.  I don't think we need to re-build this file every time we build the content-server.  I don't see the advantages of that.  Moreover, the file above need this during development.